build.gradle

```diff
@@ -10,6 +10,13 @@ repositories {
 dependencies {
     testImplementation('org.junit.jupiter:junit-jupiter:5.8.2')
 }
+
+// Pass args to arg1 like so: `./gradlew run1 -Parg1="argument value goes here" `
+task run1(type: JavaExec) {
+    mainClassName = "decide.Main"
+    classpath = sourceSets.main.runtimeClasspath
+    // args = ["$arg1"]    // Uncomment this line if you wish to pass args on cmd line
+}
```

Comment:
I don't think we need an entirely new task for this. You can instead use `gradle run --args="<path to file>"`

Reply:
ahh nice, I used the first best solution I found for passing an arg to mainClassName main function, but this works like a charm. Will revert these changes